apis/Google.Cloud.Speech.V1Beta1/Google.Cloud.Speech.V1Beta1/SpeechClient.cs

```diff
@@ -570,16 +582,27 @@ public virtual OperationsClient LongRunningOperationsClient
                 LongRunningOperationsClient,
                 callSettings);
 
+        public abstract class StreamingRecognizeStream : BidirectionalStreamingBase<StreamingRecognizeRequest, StreamingRecognizeResponse>
+        {
+        }
+
+        public virtual StreamingRecognizeStream StreamingRecognize(
```

Comment:
There's no underlying async call on the gRPC object, so I would guess (haven't checked) that this is not doing a network call.